### ISSUE TITLE
Remove duplicate boilerplate

### ISSUE THREAD

**Issue by pcallier** [Lab41/pythia#117]
Stack Exchange posts sometimes come with "Possible Duplicate" or other related boilerplate that we should try as hard as possible to strip out

**Comment by pcallier** [comment]
```
boilerplates = post_root.xpath("//*[contains(@Body, 'Duplicate')]")
for post in boilerplates:
    body_text = post.attrib['Body']
    print(body_text)
    print("-------------------")
    print(re.sub(r"<\/?blockquote>.*<\/?blockquote>(.*)",  r"\1", body_text,  flags=re.MULTILINE | re.DOTALL))
```

**Comment by dave-lab41** [comment]
Addressed by Commit #119
